src/helper/selection.js

```diff
@@ -108,30 +108,34 @@ const setItemSelection = function (item, state, fullySelected) {
         }
         _setGroupSelection(item, state, fullySelected);
     }
-    // @todo: Update toolbar state on change
     
 };
 
+/** @return {boolean} true if anything was selected */
 const selectAllItems = function () {
     const items = getAllSelectableRootItems();
+    if (items.length === 0) return false;
     
     for (let i = 0; i < items.length; i++) {
         setItemSelection(items[i], true);
```

Comment:
Just a note for the future — it would be nice to move the global paper state (`paper.project.layers`) into these methods somehow, either by passing it in, or by using an imported constant(?) That way it would easier to understand what state is being changed when looking at these functions out of context (like in a PR review :)